docs/devops-guide/manual.md

```diff
@@ -164,18 +164,34 @@ ln -s ../sites-available/jitsi.example.com jitsi.example.com
 ```
 
 ## Install Jitsi Videobridge
-Visit https://download.jitsi.org/jitsi-videobridge/linux to determine the current build number, download and unzip it:
+
+You can download binary packages for Debian/Ubuntu:
+* [stable](https://download.jitsi.org/stable/) ([instructions](https://jitsi.org/downloads/ubuntu-debian-installations-instructions/))
+* [testing](https://download.jitsi.org/testing/) ([instructions](https://jitsi.org/downloads/ubuntu-debian-installations-instructions-for-testing/))
+* [nightly](https://download.jitsi.org/unstable/) ([instructions](https://jitsi.org/downloads/ubuntu-debian-installations-instructions-nightly/))
+
+Or you can clone the Git repo and run the JVB from source using maven.
+
+Install JDK and Maven if missing:
+
 ```sh
-wget https://download.jitsi.org/jitsi-videobridge/linux/jitsi-videobridge-linux-{arch-buildnum}.zip
-unzip jitsi-videobridge-linux-{arch-buildnum}.zip
+apt-get install openjdk-8-jdk maven
 ```
 
-Install JRE if missing:
-```
-apt-get install openjdk-8-jre
+Clone source from Github repo:
+
+```sh
+cd /srv
+git clone https://github.com/jitsi/jitsi-videobridge.git
 ```
 
-_NOTE: When installing on older Debian releases keep in mind that you need JRE >= 1.7._
+Build the package:
+
+```sh
+export JVB_HOME="The path to your JVB clone."
+cd jitsi/jitsi-videobridge
+mvn install
```

Comment:
Have you tried this yourself? 
Because the github repo https://github.com/jitsi/jitsi-videobridge mentions a slightly different command.

Reply:
There is no problem using Maven install, but I am using it ``jvb.sh --host=localhost --domain= jitsi.example.com --When port = 5347 -- Secret = yoursecret1 &``` he made an error.
```
argument: apis
Exception in thread "main" java.lang.NullPointerException
	at org.jitsi.videobridge.Main.main(Main.java:78)
```
 I wonder if this command is no longer supported. What is the new startup command? I didn't find it

Reply:
Maybe ask again in the forum.
I did not try the manual method yet.

But your command also looks a bit strange:
What is `--When` for?

Try to use the exact command:
`jvb.sh --host=localhost --domain=jitsi.example.com --port=5347 --secret=YOURSECRET1 &`
without the empty spaces and bold text that you added.

Reply:
Thank you for your advice. Carry it out:
``` jvb.sh --host=localhost --domain= jitsi.example.com --port=5347 --secret=YOURSECRET1 &```
The result is the same.

I think the manual [installation document](https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-manual) is out of line with [jitsi videobridge](https://github.com/jitsi/jitsi-videobridge).

Can we make up for this part of the missing? 
I have submitted relevant questions on the [forum](https://community.jitsi.org/t/failed-to-build-jitsi-videobridge-from-source-code/70721)